session_3/blogspot/README.md

```diff
@@ -0,0 +1,24 @@
+# README
```

Comment:
Please add screenshots of your website in the README

Reply:
ok